examples/upload-media-from-local/README.md

```diff
@@ -0,0 +1,29 @@
+# Upload media from local
+
+This is a script that can upload media files to Studio from local.
+
+## How to use
+
+Before using this script, don't forget to configure your OAuth credentials (see [here](../../README.md#authorization)).
+
+Once you have them in config.json, you won't have to deal with them anymore: the example scripts automatically handle renewing expired tokens.
```

Comment:
I would delete this paragraph: it's a detail that is described in the root README, I wouldn't add it to anywhere else.

Reply:
done